routers/api/packages/rpm/rpm.go

```diff
@@ -59,6 +60,7 @@ func GetRepositoryKey(ctx *context.Context) {
 
 // Gets a pre-generated repository metadata file
 func GetRepositoryFile(ctx *context.Context) {
+	distribution := ctx.Params("distribution")
```

Comment:
Could you move the `ctx.Params("distribution")` into the `PackageFileInfo` constructor. We don't need this variable.